vnext/Shared/DevSupportManager.cpp

```diff
@@ -255,38 +232,28 @@ void DevSupportManager::StopPollingLiveReload() {
   m_cancellation_token = true;
 }
 
-std::string DevSupportManager::GetJavaScriptFromServer(
+std::pair<std::string, bool> GetJavaScriptFromServer(
```

Comment:
@acoates-ms why not return a struct here that can be augmented as needed instead of tying it to pair<string, bool> ?